remote-pay-cloud-pos-react/src/components/Payment.js

```diff
@@ -84,7 +115,14 @@ export default class Payment extends React.Component {
     componentWillReceiveProps(newProps) {
         this.payment = this.store.getPaymentByCloverId(this.payment.cloverPaymentId);
         if(newProps.refundSuccess){
-            this.setState({ showRefund: true, refundDisabled: true , refundId: this.payment.refunds[0].refundId, refundDate: this.payment.refunds[0].date});
+            let _amount = parseFloat(this.formatter.convertToFloat(this.payment.amount));
```

Comment:
It looks like 118-125 is duplicated in componentWillMount and again in componentDidMount.  I would combine this into a function.